docs/streams/developer-guide/dsl-api.html

```diff
@@ -3097,152 +3098,994 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
+                    while<br>maintaining support for both stateless and stateful operations.</p>
+                <h4>Migration Examples</h4>
+                <p>To migrate from the deprecated <code>transform</code>, <code>transformValues</code>, <code>flatTransform</code>, and
+                    <code>flatTransformValues</code> methods to the<br>Processor API (PAPI) in Kafka Streams, let&#39;s resume the
+                    previouss examples. The new <code>process</code> and <code>processValues</code><br>methods enable a more flexible
+                    and reusable approach by requiring implementations of the <code>Processor</code>
+                    or<br><code>FixedKeyProcessor</code> interfaces.</p>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Example</th>
+                            <th>Migrating from</th>
+                            <th>Migrating to</th>
+                            <th>State Type</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td><a href="#cumulative-discounts-for-a-loyalty-program-removal">Cumulative Discounts for a Loyalty Program</a>
+                            </td>
+                            <td><code>transform</code></td>
+                            <td><code>process</code></td>
+                            <td>Stateful</td>
+                        </tr>
+                        <tr>
+                            <td><a href="#categorizing-logs-by-severity-removal">Categorizing Logs by Severity</a></td>
+                            <td><code>flatTransform</code></td>
+                            <td><code>process</code></td>
+                            <td>Stateless</td>
+                        </tr>
+                        <tr>
+                            <td><a href="#traffic-radar-monitoring-car-count-removal">Traffic Radar Monitoring Car Count</a></td>
+                            <td><code>transformValues</code></td>
+                            <td><code>processValues</code></td>
+                            <td>Stateful</td>
+                        </tr>
+                        <tr>
+                            <td><a href="#replacing-slang-in-text-messages-removal">Replacing Slang in Text Messages</a></td>
+                            <td><code>flatTransformValues</code></td>
+                            <td><code>processValues</code></td>
+                            <td>Stateless</td>
+                        </tr>
+                    </tbody>
+                </table>
+                <h5 id="cumulative-discounts-for-a-loyalty-program-removal">Cumulative Discounts for a Loyalty Program</h5>
+                <pre class="line-numbers"><code class="language-java">package org.apache.kafka.streams.kstream;
```

Comment:
Remove `package` and imports ?

Especially the package is "weird" as KS does not contain the code of the example, so using the actually KS package of the DSL seems not to be ideal, but could be confusing.

Reply:
Removed.